code/messaging/src/main/java/com/adobe/marketing/mobile/MessagingPushPayload.java

```diff
@@ -95,6 +96,28 @@ static final class ActionButtons {
                 }
             };
 
+    /**
+     * Contains push payload keys to be added to Intent.extras via {@link #putDataInExtras(Intent)}
+     */
+    private static final List<String> _pushPayloadKeys =
```

Comment:
nit: It's not common to have `_` at the beginning of a variable name. Is it ok if I remove it?